test/integration/targets/vyos_smoke/tests/cli/common_utils.yaml

```diff
@@ -15,6 +15,8 @@
       - result.stdout_lines is defined
 
 # hit conditional() - used for declarative intent
+# Note, this can't be run on AWS because fully testing the vyos_interface dependencies
+# requires the ability to create and remove interfaces other than eth0
```

Comment:
Would this be possible if there were additional interface(s) on the VM?

Reply:
I believe so. Adding extra interfaces to the VM would probably also allow for the vyos_interface tests to be run on AWS since large portions of the smoke tests and interface tests are the same.

Reply:
Perfect. How many interfaces do you need?

Reply:
It looks like vyos_interface uses eth0, eth1 and eth2.

I'm curious to see if this works. Vyos_interface might have other requirements that I'm not aware of.

Reply:
Do these network interfaces require different subnets, or can they all be on the same one?

Reply:
Also, do they need to be public IPs, or can they be private?